<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDiskFragmentation</class>
 <widget class="QDialog" name="CheckDiskFragmentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="prtitionsCmboBx">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="blocksLabelTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#Blocks</string>
   </property>
  </widget>
  <widget class="QLabel" name="contiguityLabelTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Non Contiguity</string>
   </property>
  </widget>
  <widget class="QLabel" name="filesLabelTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileNumbers">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12213/2313</string>
   </property>
  </widget>
  <widget class="QLabel" name="blocksNumber">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1231231/12312</string>
   </property>
  </widget>
  <widget class="QLabel" name="contiguityPercentage">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.2%</string>
   </property>
  </widget>
  <widget class="PieChart" name="pieChart" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PieChart</class>
   <extends>QWidget</extends>
   <header>piechart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
